### DISCUSSION TITLE
Is there a way in which i can update the version of pocketbase using just the cli?

### DISCUSSION THREAD

**Discussion by sanathusk** [pocketbase/pocketbase#1980] [Q&A] [answered]
Is there a way in which i can update the version of pocketbase using just the cli say `packetbase update` running which check if the a latest version of pocketbase is available & if avaible will prompt me to download the latest version of pocketbase library?

**Comment by ganigeorgiev** [comment, marked as answer]
At the moment there is no such command.

It is in the [roadmap](https://github.com/orgs/pocketbase/projects/2/views/1?pane=issue&itemId=7205701) and eventually will be added to the prebuilt executables by repurposing the existing temporary v0.8 `upgrade` command.

**Reply by sanathusk** [reply]
> At the moment there is no such command.
> 
> It is in the [roadmap](https://github.com/orgs/pocketbase/projects/2/views/1?pane=issue&itemId=7205701) and eventually will be added to the prebuilt executables by repurposing the existing temporary v0.8 `upgrade` command.

Thanks @ganigeorgiev
